CHANGELOG.md

```diff
@@ -39,11 +39,12 @@ For detailed information on the tasks carried out during this release, please se
 
 For detailed information on the tasks carried out during this release, please see the GitHub milestone [v1.1.0.4](https://github.com/wso2/kubernetes-microgateway/milestone/10)
 
-## [v1.1.0.5] - 2022-07-27
+## [v1.1.0.5] - 2022-07-29
 
 ### Added
 
 - Add new configuration to prevent mounting an empty directory for adapter artifacts (refer to [issue](https://github.com/wso2/kubernetes-microgateway/issues/62))
 - Add new configuration to prevent mounting an empty directory for enforcer dropins JARs (refer to [issue](https://github.com/wso2/kubernetes-microgateway/issues/64))
+- Update default ingress TLS secrets (refer to [issue](https://github.com/wso2/kubernetes-microgateway/issues/68))
```

Comment:
Let's add this under the `Changed` heading
https://keepachangelog.com/en/1.0.0/

Reply:
fixed